api/Metrics/Metric.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Metrics;
+
+interface Metric
+{
+    /**
+     * Returns metric's name
+     *
+     * @access	public
+     * @return	string
+     */
+    public function getName(): string;
+
+    /**
+     * Returns metric's description
+     *
+     * @access	public
+     * @return	string
+     */
+    public function getDescription(): string;
+
+    /**
+     * Returns metric's description
```

Comment:
This comment is wrong. Maybe a copy/paste without the later update.

Reply:
Agreed, this should be `Returns metric's type` or something like that.

Reply:
Fixed